src/app/api/routes/login.py

```diff
@@ -4,21 +4,25 @@
 
 from app.api import routing, security
 from app.api.schemas import Token
-from app.db import users
+from app.db import access
 from app import config as cfg
 
 router = APIRouter()
 
 
 @router.post("/access-token", response_model=Token)
 async def login_access_token(form_data: OAuth2PasswordRequestForm = Depends()):
-
-    entry = await routing.fetch_entry(users, ('username', form_data.username))
+    entry = await routing.fetch_entry(access, [('login', form_data.username)])
```

Comment:
@florianriche your idea is to use Access as a unified authentication table for both users and devices ? At this moment it seams not working on my side...

Reply:
It was working before I started some refacto and I haven't tested since.
I will check if it is working on my side. 
Maybe you didn't restart the DB and the schema is still the same as before?

Reply:
Hmm it's working on my side.